libraries/AP_Camera/AP_RunCam.cpp

```diff
@@ -257,8 +257,16 @@ bool AP_RunCam::pre_arm_check(char *failure_msg, const uint8_t failure_msg_len)
 // OSD update loop
 void AP_RunCam::update_osd()
 {
+    bool use_armed_loop = AP::arming().is_armed();
+#if OSD_ENABLED
+    // prevent runcam stick gestures interferring with osd stick gestures
+    AP_OSD* osd = AP::osd();
```

Comment:
Could be `const`.

Reply:
We only need to do all of this work if use_armed_loop isn't already set to true...

Reply:
Fixed